stats/src/main/java/com/facebook/stats/mx/JVMStatsExporter.java

```diff
@@ -180,7 +249,7 @@ private void addStatIfMatches(String statName, Callable<Long> fetcher) {
    * 
    * @return the stat name
    */
-  private static String getStatName(ObjectName beanName, String... attributeNames) {
+  private static String getStatName(ObjectName beanName, List<String> attributeNames) {
```

Comment:
Note: If you do the other comments, you'll probably also want to allow for a `null` attribute name:

    for (String attributeName : attributeNames) {
      if (attributeName != null) {
        builder.append('.').append(attributeName);
      }
    }